main.go

```diff
@@ -123,6 +150,21 @@ func main() {
 		LeaderElectionID:       "appmesh-controller-leader-election",
 		HealthProbeBindAddress: healthProbeBindAddress,
 	})
+
+	podController := &custom.NewController{
```

Comment:
let's have constructor func for controller to be defined along with the controller struct